lib/apiservers/portlayer/restapi/handlers/scopes_handlers.go

```diff
@@ -398,12 +399,18 @@ func toEndpointConfig(e *network.Endpoint) *models.EndpointConfig {
 		ecports[i] = p.FullString()
 	}
 
-	return &models.EndpointConfig{
+	ret := models.EndpointConfig{
 		Address:   addr,
 		Container: e.ID().String(),
 		ID:        e.ID().String(),
 		Name:      e.Name(),
 		Scope:     e.Scope().Name(),
 		Ports:     ecports,
+		Gateway:   e.Gateway().String(),
```

Comment:
We have example code in containerhandler: `Direct: e.Network.Type == constants.ExternalScopeType`. How about it?